bundles/org.openhab.binding.sunsynk/src/main/java/org/openhab/binding/sunsynk/internal/handler/SunSynkAccountHandler.java

```diff
@@ -0,0 +1,134 @@
+import org.openhab.binding.sunsynk.internal.config.SunSynkAccountConfig;
+import org.openhab.core.thing.Bridge;
+import org.openhab.core.thing.ChannelUID;
+import org.openhab.core.thing.ThingStatus;
+import org.openhab.core.thing.ThingStatusDetail;
+import org.openhab.core.thing.binding.BaseBridgeHandler;
+import org.openhab.core.types.Command;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * The {@link SunSynkAccountHandler} is responsible for handling the SunSynk Account Bridge
+ *
+ *
+ * @author Lee Charlton - Initial contribution
+ */
+
+@NonNullByDefault
+public class SunSynkAccountHandler extends BaseBridgeHandler {
+    private final Logger logger = LoggerFactory.getLogger(SunSynkAccountHandler.class);
+    private AccountController sunAccount = new AccountController();
+    private @Nullable ScheduledFuture<?> discoverApiKeyJob;
+
+    public SunSynkAccountHandler(Bridge bridge) {
+        super(bridge);
+    }
+
+    @Override
+    public void handleCommand(ChannelUID channelUID, Command command) {
+    }
+
+    @Override
+    public void initialize() {
+        updateStatus(ThingStatus.UNKNOWN);
+        logger.debug("SunSynk Handler Intialised attempting to retrieve configuration");
+        startDiscoverApiKeyJob();
+    }
+
+    private void startDiscoverApiKeyJob() {
+        if (discoverApiKeyJob == null || discoverApiKeyJob.isCancelled()) {
+            Runnable runnable = new Runnable() {
+                @Override
+                public void run() {
+                    logger.debug("Starting account discovery job.");
+                    configAccount();
+                    logger.debug("Done account discovery job.");
+                }
+            };
+            discoverApiKeyJob = scheduler.schedule(runnable, 1, TimeUnit.SECONDS);
+        }
+    }
+
+    public void setBridgeOnline() {
+        updateStatus(ThingStatus.ONLINE);
+    }
+
+    public void setBridgeOffline() {
+        updateStatus(ThingStatus.OFFLINE);
+    }
+
+    public List<Inverter> getInvertersFromSunSynk() {
+        logger.debug("Attempting to find inverters tied to account");
+        ArrayList<Inverter> inverters = new ArrayList<>();
+        try {
+            inverters = sunAccount.getDetails();
+        } catch (SunSynkInverterDiscoveryException e) {
+            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.COMMUNICATION_ERROR,
+                    "Error attempting to find inverters registered to account");
+        }
+        return inverters;
+    }
+
+    @Override
+    public void dispose() {
+        logger.debug("Disposing sunsynk bridge handler.");
+        if (discoverApiKeyJob != null && !discoverApiKeyJob.isCancelled()) {
+            discoverApiKeyJob.cancel(true);
+            discoverApiKeyJob = null;
+        }
+    }
+
+    public void configAccount() {
+        SunSynkAccountConfig accountConfig = getConfigAs(SunSynkAccountConfig.class);
```

Comment:
Consider to check confige.email and config.password for .isBlank() and if eitehr ons is blank, set Thing status to ThingStatusDetail.CONFIG_ERROR

Reply:
Done